docs/topics/gradle.md

```diff
@@ -316,10 +293,51 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
 >
 {type="note"}
 
+Additionally, you can use the `kotlin-test` dependency in any shared or platform-specific source set.
+
+For Kotlin/JVM Gradle uses JUnit 4 by default; therefore, the `kotlin('test')` dependency resolves to JUnit 4.
+
+You can choose JUnit 5 or TestNG by calling 
+[`useJUnitPlatform()`]( https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform) 
+or [`useTestNG()`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useTestNG).
+The following example is for a JVM project:
+
+<tabs>
+
+```groovy
+dependencies {
+    testImplementation 'org.jetbrains.kotlin:kotlin-test'
+}
+
+test {
+    useTestNG()
+}
+```
+
+```kotlin
+dependencies {
+    testImplementation(kotlin("test"))
+}
+
+tasks {
+    test {
+        useJUnitPlatform()
+    }
+}
+```
+
+</tabs>
+
+See [the tutorial on how to test code using JUnit in JVM](#jvm-test-using-junit.md).
+
+If the standard variants described above don't cover you needs, you can disable the automatic selection
```

Comment:
In general, I'd start this sentence from a more exact case description such as: "If you need to use a different JVM test framework, you can ... and add this framework as a dependency ..."

Reply:
I fixed but I'm afraid that the new sentence is too long and complex.